### PR TITLE
refactoring(footer component): footer are ready for test

### PR DESCRIPTION
**Pull request by DiLiUA**
- move footer component from Shared folder;
  - change styles and markup for responsive design;

### CODE REVIEW

File: src/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<footer id="footer">
```
Comment (dpereverzev) [inline src/footer/footer.component.html]: id?

File: src/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="footer-columns col-md-3">
+          <h3 class="title">Help</h3>
+          <a href="#" routerLink="/help-support" class="footer-link support">Help & Support</a>
+          <a href="#" routerLink="/faq" class="footer-link faq">FAQ</a>
+          <a href="#" routerLink="/contact" class="footer-link contact">Contact Us</a>
+        </div>
+
+        <div class="footer-columns col-md-3">
+          <h3 class="title">About LiveStarter</h3>
+          <a href="#" routerLink="/how-it-works" class="footer-link how-it-was">How It Works</a>
```
Comment (dpereverzev) [inline src/footer/footer.component.html]: you don't need href='#' with routerLInk

File: src/footer/footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
+@import "../variables.scss";
+
+#footer {
```
Comment (dpereverzev) [inline src/footer/footer.component.scss]: try to avoid id's in styles

File: src/footer/footer.component.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	padding-top: 30px;
+	padding-bottom: 30px;
+}
+
+.footer-columns {
+  padding: 10px;
+}
+
+.title {
+	color: white;
```
Comment (dpereverzev) [inline src/footer/footer.component.scss]: IMHO: use variable color $white or use only hex #fff;

File: src/footer/footer.component.scss [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+	text-align: center;
+	background-color: #1a2930;
+	color: #53636b;
+	font-size: 15px;
+	font-weight: 400;
+	line-height: 34px;
+	padding: 10px;
+}
+
+@media screen and (max-width: 960px) {
```
Comment (dpereverzev) [inline src/footer/footer.component.scss]: i think you should create mixin with your breakpoints and use it

**Review by dpereverzev — CHANGES_REQUESTED** [review]

**Review by Ugmaxie — APPROVED** [review]

**Pull request merged**